rel-eng/tito.props

```diff
@@ -9,3 +9,12 @@ blacklist =
 [katello-nightly-rhel7]
 disttag = .el7
 blacklist =
+
+[katello-nightly-fedora20]
+disttag = .fc20
+whitelist = katello-repos
+
+[katello-nightly-fedora21]
+disttag = .fc21
+whitelist = katello-repos
```

Comment:
The whitelist/blacklist here is for the base package (not subpackages).  So here we say build 'katello-repos' for f21, and in the comps we only allow katello-client-repos into the actual yum repo.  Sure there is a rpm build on koji that is never used but the only way around that would be to split these two rpms up into separate source rpms.